isic_archive/tasks/image.py

```diff
@@ -175,7 +177,7 @@ def generateSuperpixels(self, imageId):
 @app.task(bind=True)
 def generateLargeImage(self, imageId):
     imageItem = Image().load(id=imageId, force=True)
-    imageFile = Image().strippedFile(imageItem)
+    imageFile = Image().originalFile(imageItem)
```

Comment:
I'm unsure about whether this change matters.

Reply:
I think it's better this way, but I'm not critically worried about the old large_image files right now.

Reply:
I think it's better this way, but I'm not critically worried about the old large_image files right now.